backend/dal/Services/Admin/Concrete/UserService.cs

```diff
@@ -87,6 +87,24 @@ public override void Remove(User entity)
             this.Context.Entry(user).CurrentValues.SetValues(entity);
             base.Remove(user);
         }
+
+        /// <summary>
+        /// Get all the access requests that users have submitted to the system
+        /// </summary>
+        /// <param name="page"></param>
+        /// <param name="quantity"></param>
+        /// <param name="sort"></param>
+        /// <param name="isGranted"></param>
+        public Paged<AccessRequest> GetAccessRequestsNoTracking(int page = 1, int quantity = 10, string sort = null, bool? isGranted = null)
+        {
+            var query = this.Context.AccessRequests.AsNoTracking();
```

Comment:
Might want to handle bad argument values like `page= <1`, etc.

Reply:
Agreed.